app/ui/legacy/src/main/res/layout/recipient_alternate_item.xml

```diff
@@ -143,6 +143,15 @@
             tools:visibility="visible"
             />
 
+        <ImageView
```

Comment:
The clickable area is too small. The recommendation for Android is an area of at least 48dp x 48dp.